Black-Market/Black-Market/Helpers/UIConstants.swift

```diff
@@ -11,10 +11,17 @@ struct UIConstants {
         static let cornerRadius: CGFloat = 8
         static let maxHeight: CGFloat = 44
         static let lineWidth: CGFloat = 1
+        static let spacing: CGFloat = 16
+        static let padding: CGFloat = 28
+        static let spacerHeight: CGFloat = 20
     }
     
     enum MarketTexfield {
         static let size: CGFloat = 14
         static let lineWidth: CGFloat = 2
     }
+    
+    enum SignUpView {
```

Comment:
could this be in an extension of UIConstants in the file of interest? This will prevent this file to get bigger and bigger and keep constants private to where they are used only

Reply:
Done